jsons/1.12.2-rel.json

```diff
@@ -38,9 +38,9 @@
       "clientreq":true
     },
     {
-      "name": "jline:jline:2.13",
+      "name": "org.jline:jline:3.5.1",
       "url" : "http://files.minecraftforge.net/maven/",
-      "checksums" : [ "2d9530d0a25daffaffda7c35037b046b627bb171" ],
+      "checksums" : [ "51800e9d7a13608894a5a28eed0f5c7fa2f300fb" ],
       "serverreq":true,
       "clientreq":false
     },
```

Comment:
You need [JNA here](https://github.com/MinecraftForge/MinecraftForge/pull/3965/files#diff-84f10d880ba79d448af943c7cdec3cefR62) to make sure it is available on the server. Minecraft only bundles it on the client.

Reply:
Hm ok. Based on `jsons/1.12.1.json` I thought it was included on both client and server already.
Is there a better way to tell what is included, short of pulling open the vanilla server jar?

Reply:
Working fine for me. On which OS are you testing?

Reply:
Windows with Cygwin terminal.

Reply:
Huh, well, can you test in a regular Windows terminal?

Reply:
I'm going to wipe out my workspace and try again, I suspect something is up with it because I have been working on a few different branches and switching between.

Reply:
Still have the same issue in cygwin (no issue before this commit).
I tested the plain windows terminal and it has no issues at all.

Reply:
This might be an issue in the newer JLine version. I will take a look when I have time, but not sure if this is a big problem until then, I don't think many people run the server in a Cygwin terminal.

Reply:
Cygwin is generally pretty weird, which is one of the reasons I picked it for testing.
Having the terminal keep the server up until it receives input is big an issue because it breaks scripts that auto-restart servers when they crash. There was a problem with that before.